### DISCUSSION TITLE
DataTable Save Locally

### DISCUSSION THREAD

**Discussion by joelowj** [datapane/datapane#101] [General]
Hi, 

I am just asking if it is possible to bring back DataTable without forcing us to update the data to the public server. There are some constraints around where some data can leave the firm / country. This means that I am totally cut off from being able to use DataTable.

**Comment by mands** [comment]
Hi there - sorry for the wait on this, there were some technical issues, however happy to announce that DataTable is now supported again when saving locally from Datapane version 0.11.6 onwards.
